website/content/partials/provisioner/ansible-local/Config-not-required.mdx

```diff
@@ -11,20 +11,20 @@
 - `extra_arguments` ([]string) - Extra arguments to pass to Ansible.
   These arguments _will not_ be passed through a shell and arguments should
   not be quoted. Usage example:
-  
```

Comment:
These file here are auto-generated and the generator does its best, but sometimes it's bad with trailing spaces etc. If you run `make generate` here (which should have been run by circle ? 🤔) I think it will remove those.

Reply:
nvm did that !